content/english/sql_workshop/activity3/_activity3.md

```diff
@@ -7,24 +7,54 @@ difficulty: "Beginner"
 draft: false
 translationKey: "sql_activity_3"
 ---
-<!-- Links for javascript and CSS needed for drop down logic 
+<!-- Links for javascript and CSS needed for drop down logic -->
 <link rel="stylesheet" href="../_activity3.css" type="text/css"></link>
-<script type="text/javascript" src="../_activity3.js"></script> -->
+<script type="text/javascript" src="../alasql.js"></script>
+<script type="text/javascript" src="../_activity3.js"></script>
 
-#### You have successfully arrived on the Planet of Fun! The Galactic Federation has sent you a database about this planet.
-![Planet_DB](../assets/planet_db.PNG)
+<script>
+alasql("CREATE TABLE planet(city_number INT, name STRING, population INT, favorite_food STRING)");
+alasql("INSERT INTO planet VALUES (1,'LEGO City',1500,'Pizza')");
+alasql("INSERT INTO planet VALUES (2,'Playground City',1000,'Pizza')");
+alasql("INSERT INTO planet VALUES (3,'City of Games',6000,'Hotdog')");
+alasql("INSERT INTO planet VALUES (4,'Fun Capital',9500,'Hamburger')");
+alasql("INSERT INTO planet VALUES (5,'Seriously Fun',2000,'Taco')");
 
-## Task 3: Talk to the inhabitants
 
-You need to talk to the inhabitants of the Planet of Fun to discover what the problem is. 
-Find out how many cities there are on the planet. You will learn how to use the COUNT command!
 
-**What the command does:**
 
+</script>
+# Congratulations!
+You have successfully arrived on the Planet of Fun! The Galactic Federation has sent you a database about this planet.
 
-**Example:** 
+![db](../assets/planet_db.PNG)
 
+## Task 3: Talk to the inhabitants 
+You need to talk to the inhabitants of the Planet of Fun to discover what the problem is. Find out how many cities there are on the planet using the COUNT command!
 
-<!-- Activity goes here -->
+When we use the SELECT * FROM planet; command, this table is displayed. But in order to check the number of cities, we can use the COUNT command.
+
+![count](../assets/count.png)
+
+### Question: Using COUNT, what command will display the number of cities on the Planet of Fun?
+{{< rawhtml >}}
+	
+	<div class="terminal_div" id="terminal_div">
+		<img class="terminal" src="../assets/Terminal.png" alt="Terminal_screen" style="pointer-events:none">
+		<div class="text-area">
+			<textarea id="commands" placeholder="Enter command here!" style="resize: none"></textarea>
+		</div>
+	</div>
+
+	
+	<button class="button button1" onclick="sql()">	Enter </button>
```

Comment:
I just realized that this will be interesting when we go to localize this workshop... Maybe after everything is done, we should do a pass through and identify the places where translations are required from all the HTML code that's here.